app/models/post.rb

```diff
@@ -1,3 +1,5 @@
 class Post < ApplicationRecord
-  belongs_to :author
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  has_many :comments, dependent: :destroy
```

Comment:
- [ ] please kindly ensure the Post model has a method that returns the 5 most recent comments for a given post.